dvc/lock.py

```diff
@@ -68,6 +68,10 @@ def unlock(self):
         self._lock.close()
         self._lock = None
 
+    @property
+    def is_locked(self):
```

Comment:
For the record, flufl has that method by default, so don't need to add it there.